chocolatey/choco
# (#158) Restore `--source` tab completion for list command

## Description Of Changes

- Add back `--source` for `list` command

## Motivation and Context

We added the option back to allow it to be used for alternative sources, it needs to be provided by tab completion as well.

## Testing

- Load the tab completion / profile module in a PS session
- Type `choco list ` (note: trailing space) and press tab a few times
- `--source=''` should show up in the available completions

### Operating Systems Testing

Win11

## Change Types Made



## Change Checklist


## Related Issue

#158
